tools/copycat/ccsra.c

```diff
@@ -606,24 +591,7 @@ rc_t CC CCNodeSraDirFilePhysicalSize (const CCNodeSraDir *self,
                                       const char *path_fmt,
                                       va_list args)
 {
-    char path[4096];
```

Comment:
`CCNodeSraDirFilePhysicalSize` seem to do the same thing as `CCNodeSraDirFileSize`